quickstarts/maven/micronaut/src/test/java/org/eclipse/jkube/quickstart/micronaut/custom/SimpleTest.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) 2019 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at:
+ *
+ *     https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.jkube.quickstart.micronaut.custom;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
+import org.junit.jupiter.api.Test;
+
+import jakarta.inject.Inject;
+
+@MicronautTest
+class SimpleTest {
```

Comment:
Maybe we should not add a test since it's just a quickstart? What do you think?

Reply:
I added the unit to confirm that the Micronaut 3.x application exposes the endpoints. I know it doesn't add significant value, but it does confirm that Micronaut 3.x is working.

Reply:
ohk, No worries. I guess we can keep it :-)